server/src/main/java/org/opensearch/repositories/Repository.java

```diff
@@ -207,10 +207,15 @@ default void deleteSnapshotsAndReleaseLockFiles(
     long getRestoreThrottleTimeInNanos();
 
     /**
-     * Returns restore throttle time in nanoseconds
+     * Returns upload throttle time in nanoseconds
      */
     long getRemoteUploadThrottleTimeInNanos();
 
+    /**
+     * Returns low priority upload throttle time in nanoseconds
+     */
+    long getLowPriorityRemoteUploadThrottleTimeInNanos();
```

Comment:
Will this break implementations of `Repository`?

Reply:
Yes, the other option is to have a default implementation . But most of repo extend `BlobRepository`, so they should be fine . However CCR plugin will break like before : https://github.com/opensearch-project/cross-cluster-replication/pull/1111

Reply:
Right. Also, there could be other repository implementations. Having default implementation is required IMO.

Reply:
Makes sense . Addressed .